src/main/java/com/minecolonies/coremod/commands/colonycommands/CommandSetAbandoned.java

```diff
@@ -36,8 +35,8 @@ public int onExecute(final CommandContext<CommandSource> context)
         }
 
         boolean addOfficer = false;
-        if (sender != null && (colony.getPermissions().getRank((PlayerEntity) sender) == Rank.OFFICER
-                                 || colony.getPermissions().getRank((PlayerEntity) sender) == Rank.OWNER))
+        if (sender != null && (colony.getPermissions().getRank((PlayerEntity) sender) == colony.getPermissions().getRankOfficer()
```

Comment:
All the static rank checks should probably be replaced by permission checks now

Reply:
I'm also not 100% happy with the colony.getPermissions().getRankOfficer(), maybe if we have constants, we could do colony.getPermissions().getRank(RankConstants.OFFICER) or so